### PR TITLE
Avalonia - Save Manager

### PR DESCRIPTION
**Pull request by emmauss**
Adds a save manager for the avalonia ui for managing user saves. Currently only supports viewing and deleting saves and restoring lost accounts using existing saves.
Screenshot from 2022-07-29 11-58-46

### CODE REVIEW

**Comment by jduncanator** [comment]
Might I suggest providing screenshots in the PR body for UI-related changes, especially for new functionality?

**Comment by sakuka47** [comment]
Screenshot_20220718-201249_TeamViewer

**Comment by emmauss** [comment]
Added an option to recover accounts that have been lost due to the profiles json being corrupted or modified. The options scans for saves with no available user linked to it, and creates a new user with the user id that's linked to the save.

File: Ryujinx.Ava/Assets/Locales/en_US.json [hunk 1]
```diff
@@ -592,5 +592,16 @@
   "RyujinxUpdaterMessage": "Do you want to update Ryujinx to the latest version?",
   "SettingsTabHotkeysVolumeUpHotkey": "Increase Volume:",
   "SettingsTabHotkeysVolumeDownHotkey": "Decrease Volume:",
-  "VolumeShort": "Vol"
+  "VolumeShort": "Vol".
```
Comment (marysaka) [inline Ryujinx.Ava/Assets/Locales/en_US.json:595]: typo
```suggestion
  "VolumeShort": "Vol",
```

**Review by marysaka — COMMENTED** [review]
Untested, but found a typo.

**Review by AcK77 — APPROVED** [review]

**Review by marysaka — APPROVED** [review]

**Pull request merged**